stdlib/experimental/dynamic/dynamic.go

```diff
@@ -2,13 +2,19 @@ package dynamic
 
 import (
 	"context"
+
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/internal/errors"
 	"github.com/influxdata/flux/interpreter"
 	"github.com/influxdata/flux/runtime"
+	"github.com/influxdata/flux/semantic"
 	"github.com/influxdata/flux/values"
 )
 
 func init() {
 	runtime.RegisterPackageValue("experimental/dynamic", "dynamic", dynamicConv)
+	runtime.RegisterPackageValue("experimental/dynamic", "asArray", asArray)
+	runtime.RegisterPackageValue("experimental/dynamic", "_equal", equal)
```

Comment:
It seems odd to me that there's no op spec or procedure spec for these new functions. I wonder how the planner handles the nodes without them? Do we know how these show up in a planner graph?

Reply:
I think those are only used for functions that operate on tables, but I could be totally wrong.

My intuition was that they are the bits that connect the stream of operations together. Since dynamic cannot be added to tables, it can't appear in table streams, therefore no op spec or what have you. All this work happens in the interpreter, which I guess is handled differently.

Reply:
Yes, you're right @onelson. No need for an OpSpec for these, since they operate only on scalar values. They are just like the function in the `strings` package for example, `strings.substr`, etc.

Reply:
Ah, I see. Just a misunderstanding on my part then. I don't think I've ever added a function that wasn't manipulating table streams, so that stuck out to me. But it makes sense that we wouldn't need those for functions that manipulative individual scalars/collections.